src/core/simple/src/helpers/orders.ts

```diff
@@ -10,8 +10,7 @@ export const request = order =>
         resolve(order)
       }
     })
-  ).then(order => {
-    //@ts-ignore
+  ).then((order: { id: number | null }) => {
```

Comment:
Hi, thanks for your request. Have you checked the type of order id? I'm not sure that it's number. As I can recognize it's a string